ui/app/components/clients/page/counts.ts

```diff
@@ -162,6 +164,22 @@ export default class ClientsCountsPageComponent extends Component<Args> {
     return activity?.total;
   }
 
+  get showSecretsSync(): boolean {
```

Comment:
since this value is only needed on one component, we don't need to compute this in the route to add to the model. better off calculating it closer to where we actually use the value, where we already have enough context to compute it. in other words, by calculating the value here we can simply pass it straight into the `<NavBar />` which is the only place where it's used. 😀